exosphere-shared/src/docker-helper.ls

```diff
@@ -1,146 +1,49 @@
 require! {
-  \child_process
   \prelude-ls : {any}
-  \observable-process : ObservableProcess
-  \os
+  'dockerode' : Docker
+  'wait' : {wait}
 }
 
+docker = new Docker
 
+# Helper class used to manage Docker processes not started by docker-compose
 class DockerHelper
 
-  @build-all-images = ({write}, cb) ->
-    new ObservableProcess('docker-compose build'
-                          stdout: {write}
-                          stderr: {write})
-      ..on 'ended', cb
+  @start-container = ({Image, name, HostConfig}, done) ->
+    DockerHelper._container-is-running name, (err, is-running) ->
+      | err => done err
+      | is-running => done!
+      | otherwise =>
+        docker.create-container {Image, name, HostConfig}, (err, container) -> 
+          | err => done err
+          container.start (err, data) ->
+            | err => done err
+            wait 2_000, done
```

Comment:
I agree, I just put this in as a temporary fix. In the future we want to add multiple listeners to this process as we discussed here: https://github.com/Originate/exosphere-sdk/issues/280

But I agree, I am uncomfortable with this implementation and am open to any other ideas.